src/dummy/codes.json

```diff
@@ -3,70 +3,153 @@
         "id": 1,
         "words": [
```

Comment:
If you only consider the Json you are right. But because our implementation treats every word explicitly and is very focused on the "words" I think `words` is a good name in this domain.